client/src/App.jsx

```diff
@@ -129,6 +134,11 @@ const ReviewsDiv = styled.div`
 class App extends React.Component {
   constructor(props) {
     super(props);
+
+    var today = new Date();
+    var oneMonthFromToday = new Date(today);
+    oneMonthFromToday.setDate(today.getDate() + 30);
```

Comment:
Does this work during a leap year in February?

Reply:
Nope it doesn't even work today lol. Fixing it now!! You mind if I just fix it with my next PR?